content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -93,6 +96,196 @@ service:
+
+To accurately track the appropriate metadata in Datadog, run the OpenTelemetry Collector in agent mode on each of the Kubernetes nodes.
+
+When deploying the OpenTelemetry Collector as a daemonset, refer to [the example configuration below](#opentelemetry-kubernetes-example-collector-configuration) as a guide.
+
+On the application container, use the downward API to pull the host IP. The application container needs an environment variable that points to `status.hostIP`. The OpenTelemetry Application SDKs expects this to be named `OTEL_EXPORTER_OTLP_ENDPOINT`. Use the [below example snippet](#opentelemetry-kubernetes-example-application-configuration) as a guide.
+
+##### Example Kubernetes OpenTelemetry Collector configuration
+
+A full example k8s manifest for deploying the OpenTelemetry Collector as both daemonset and standalone collector [can be found here][14]. Depending on your environment this example may be modified, however the important sections to note specific to Datadog are as follows.
+
+1. The example demonstrates deploying the OpenTelemetry Collectors in ["agent" mode via daemonset][15], which collect relevant k8s node and pod specific metadata, and then forward telemetry data to an OpenTelemetry Collector in ["standalone collector" mode][16]. This OpenTelemetry Collector in "standalone collector" mode then exports to the Datadog backend. A diagram of this deployment model [can be found here][17].
+
+2. For OpenTelemetry Collectors deployed as agent via daemonset, in the Daemonset, `spec.containers.env` should use the downward API to capture `status.podIP` and add it as part of the `OTEL_RESOURCE` environment variable. This is used by the OpenTelemetry Collector's `resourcedetection` and `k8s_tagger` processors, which should be included along with a `batch` processor and added to the `traces` pipeline.
+
+- In the DaemonSet's `spec.containers.env` section
+
+```
+  # ...
+  env:
+     # Get pod ip so that k8s_tagger can tag resources
+    - name: POD_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: status.podIP
+      # This is picked up by the resource detector
+    - name: OTEL_RESOURCE
+      value: "k8s.pod.ip=$(POD_IP)"
+  # ...
+```
+
+- In the `otel-agent-conf` ConfigMap's `data.otel-agent-config` `processors` section
+
+```
+  # ...
+  # The resource detector injects the pod IP
+  # to every metric so that the k8s_tagger can
+  # fetch information afterwards.
+  resourcedetection:
+    detectors: [env]
+    timeout: 5s
+    override: false
+  # The k8s_tagger in the Agent is in passthrough mode
+  # so that it only tags with the minimal info for the
+  # collector k8s_tagger to complete
+  k8s_tagger:
+    passthrough: true
+  batch:
+  # ...
+```
+
+- In the `otel-agent-conf` ConfigMap's `data.otel-agent-config` `service.pipelines.traces` section
+
+```
+  # ...
+  # resourcedetection must come before k8s_tagger
+  processors: [resourcedetection, k8s_tagger, batch]
+  # ...
+```
```

Comment:
For the code fence blocks above, you'll want to intent them 4 (I think) spaces so that they are all part of step 2. Would need a docs staging build to know for sure that will work (can only do that from a branch not a fork, or from a local build).

Reply:
Also, these are yaml, right? Can we add `yaml` after the three ticks, for some formatting on the website?

Reply:
yuup, updated!